docs/tutorial/force_calibration.rst

```diff
@@ -212,3 +212,98 @@ If we compare the different fits, we can see that the Lorentzian model with diod
 While this is true for this particular dataset, no general statement can be made about the bias of fitting a Lorentzian rather than the hydrodynamically correct power spectrum.
 If low bias is desired, one should use the hydrodynamically correct model.
 On regular sensors, it is best to fit the hydrodynamically correct model with the diode model enabled.
+
+Active Calibration
+------------------
+
+Sometimes, more accuracy is required.
+One way to achieve this is with active force calibration.
+By providing a known perturbation to the bead, active calibration uses fewer assumptions than passive calibration.
+
+When performing passive calibration, we rely on knowledge of the drag coefficient to obtain a trap stiffness and positional calibration.
+This theoretical drag coefficient depends on parameters that are only known with limited precision.
+These parameters are the diameter of the bead and the viscosity.
+This viscosity in turn depends strongly on the local temperature around the bead, which is typically also poorly known.
+
+When performing active calibration, either the nanostage or trap is oscillated with a specific amplitude.
+This results in a visible driving peak at that particular frequency in the power spectrum.
+If we choose the windowing to include a full number of periods of the driving input, then the resulting driving peak will end up in a single data point in the frequency spectrum.
+
+.. image:: driving_input.png
+
+The physical spectrum is then given by a thermal part (like before) and an active part:
+
+.. math::
+
+    P_{thermal}(f) = \frac{D}{\pi ^ 2 \left(f^2 + f_c ^ 2\right)}
+
+    P_{active}(f) = \frac{A^2}{2\left(1 + \frac{f_c^2}{f_{drive}^2}\right)} \delta(f - f_{drive})
+
+    P(f) = P_{thermal}(f) + P_{active}(f)
+
+Since we know the driving amplitude, we know how the bead reacts to the driving motion and we can observe this response in the PSD, we can use this relation to determine the positional calibration.
+
+If we use the basic Lorentzian model, then the theoretical power (integral over the delta spike) corresponding to the driving input is given by :cite:`tolic2006calibration`:
+
+.. math::
+
+    W_{theory} = \frac{A^2}{2\left(1 + \frac{f_c^2}{f_{drive}^2}\right)}
+
+Subtracting the thermal part of the spectrum, we can determine the same quantity experimentally.
+
+.. math::
+
+    W_{experiment} = \left(P^{volt}(f_{drive}) - P^{volt}_{thermal}(f_{drive})\right) \Delta f
+
+Here the thermal contribution that needs to be subtracted is obtained from fitting the thermal part of the spectrum using the passive calibration procedure from before.
+The desired positional calibration is then:
+
+.. math::
+
+    \beta = \sqrt{\frac{W_{theory}}{W_{experiment}}}
+
```

Comment:
The expressions of W_theory and W_experiment now look identical, as all the variables have exactly the same name. So it looks as if this ratio is always 1. Maybe explaining the superscript 'volt' will make more clear what this expression means exactly.

Reply:
I could add a superscript to the `W`s that says `volt` or `physical`.

I'm actually beginning to doubt a little whether it shouldn't be `sensor` vs `physical`, because the integrated PSD is actually in volts squared, and calling it Volt may be a little confusing. Then again, after the previous change, it should be clear what the superscript means now.

What do you think?

Reply:
I actually find volt vs physical confusing, as Volt is a proper unit, but physical is not.
It would be clearest to give the units explicitly. How about using the labels `sensor` and `physical `and then specify the specific units in the text?

Reply:
I agree. Poor choice on my part there.

How about `measured` and `physical` and we add units to the equations most relevant to the positional calibration.

I've just pushed a version with those changes.